docs/development/architecture.md

```
@@ -257,20 +257,19 @@ CMakeLists.txt (main)
 
 ## Translation System
 
-DevTools supports multiple languages using Qt Linguist:
+DevTools supports multiple languages using Qt translation tools:
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)